README.md

```diff
@@ -51,8 +51,9 @@ lists:
 | Users & Operators | frog@lists.frrouting.org     |
 | Announcements     | announce@lists.frrouting.org |
 
-For chat, we currently use [Slack](https://frrouting.slack.com). Please email
-the mailing list to request an invite as we do not issue automatic invites.
+For chat, we currently use [Slack](https://frrouting.slack.com).
+You can join the slack by clicking the "Slack" link under the
```

Comment:
I would say `Slack` is a proper noun, so it should be capitalized. Other than that, it looks fine to me.

Reply:
It was a typo...